projects/igniteui-angular/src/lib/grids/cell.component.html

```diff
@@ -1,4 +1,5 @@
 <ng-template #defaultCell>
+    <igx-chip *ngIf="displayPinnedChip" class="igx-grid__td--pinned-chip" [disabled]="true" [displayDensity]="'compact'">Pinned</igx-chip>
```

Comment:
I am thinking if the chip should respect the grid density rather than always being "compact" because it appears in body rows that change the row/cell height, what do you think.